Library/Homebrew/utils/formatter.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/tty"
+
+module Formatter
+  module_function
+
+  def arrow(string, color: nil)
+    prefix("==>", string, color)
+  end
+
+  def headline(string, color: nil)
+    arrow("#{Tty.bold}#{string}#{Tty.reset}", color: color)
+  end
+
+  def success(string, label: nil)
+    label(label, string, :green)
+  end
+
+  def warning(string, label: nil)
+    label(label, string, :yellow)
+  end
+
+  def error(string, label: nil)
```

Comment:
I kinda agree. How do you suggest we proceed? The following works for me:

```ruby
Formatter.error(message) # default label "Error"
Formatter.error(message, label: "READ THIS") # custom label
```

But I don't know how to deal with the case where there's neither a label nor a message, but just a string that should be styled like the error label. Should that be a separate method?